RecoTauTag/RecoTau/plugins/rerunMVAIsolationOnMiniAOD_Phase2.cc

```diff
@@ -0,0 +1,226 @@
+#include "FWCore/Framework/interface/EDAnalyzer.h"
```

Comment:
please move this file to RecoTauTag/RecoTau/test/

Reply:
When rerunMVAIsolationOnMiniAOD_Phase2.cc was in /test it was not compiling, but when it was in /plugins it did. Upon inspection it seems there is a special block in the BuildFile in the test directory for rerunMVAIsolationOnMiniAOD.cc:
https://github.com/cms-tau-pog/cmssw/blob/8c1791c61fce4b13fffae40cc83a2c75dae3469e/RecoTauTag/RecoTau/test/BuildFile.xml#L5-L18
and if add a similiar block to the BuildFile for the _Phase2.cc version it appears to compile properly.

Reply:
Alright, please add these as well in this case. Thanks!